portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase

```diff
@@ -1833,10 +1828,9 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-174431 TC-9. Verify there's a confirmation dialog when a friendly URL collision exists and Administrator tries to edit an existing Site Template page."
-	@ignore = "true"
 	@priority = 4
 	test ViewConfirmationDialogOfEditExistingTemplatePageURLCollision {
-		property custom.properties = "feature.flag.LPS-174431=true";
+		property custom.properties = "feature.flag.LPS-174417=true";
```

Comment:
The current test point is covered by CPSitetemplates#ViewAlertAfterEditExistingTemplatePageURLCollision.
Please remove this test.